SolidBeginnerWorkshopNotes.md

```diff
@@ -0,0 +1,149 @@
+
+# About Solid
+
+## About the Solid Project
+
+- Decentralized web
+- Project started by Sir Tim Berners-Lee; inventor of the World Wide
+    Web
+- *Solid is an open standard for structuring data, digital identities,
+    and applications on the Web.*
+- Goal: for people to have more agency over their data
+    - **True data ownership** - Users should have the freedom to
+        choose where their data resides and who is allowed to access it.
+    - **Fully interoperable standards** - Solid uses a common, shared
+        way of describing things and their relationships to one another
+        that different applications can understand.
+        - This gives Solid the unique ability to allow different
+            applications to work with the same data.
+        - Prevents application vendor lock-in
+    - **Granular access --** Solid's authentication and authorization
+        mechanisms allow one to determine which people and applications
+        can access their data
+
+## Solid in a nutshell
+
+- **Solid is a specification** consisting of several Technical
+    Reports, including the Solid Protocol:
+    <https://solidproject.org/TR/protocol>
+    - *The specifications in the ecosystem describe how Solid servers
+        and clients can be interoperable by using Web communication
+        protocols, global identifiers, authentication and authorization
+        mechanisms, data formats and shapes, and query interfaces.*
+
+- **Solid Pod: Pods are where you store your data**.
+    - Any kind of data can be stored in a Solid Pod
+        - Structured data, files
+    - Linked Data which allows different applications to work with the
+        same data
+        - *Linked Data is a set of design principles for sharing
+            machine-readable interlinked data on the Web.*
+    - Once data is in your Pod, you control access to it.
+
+- **Pod Provider:**
+    - You can get a Pod from a Pod Provider, or you can self-host your
+        Pod. (You can also implement a new Pod Provider if you like
+        implementing from specifications!)
+    - <https://solidproject.org/users/get-a-pod>
+        - Many Pod Providers to choose from! Some community-based,
+            some by companies.
+        - A Pod Provider will usually advertise the underlying cloud
+            infrastructure it is using, and the hosting location. For
+            example, Inrupt Pod Spaces uses Amazon for hosting and the
+            hosting location is Germany.
+
+- **Solid applications:**
+    - Any type of application can be built using the Solid Protocol.
+    - Applications can make requests to retrieve data from your Pod (via your Pod Provider), and also to store data on your Pod. This is the essential part of how Solid works. 
+        - Keep in mind that you ALWAYS control access of your files on your Pod, so you can revoke access to an application at any time. 
+    - Example of applications: <https://solidproject.org/apps>
```

Comment:
A more comprehensive catalog is at https://github.com/solid/catalog. https://solid-catalog.jeswr.org is another frontend for that data